src/utils/generator.js

```diff
@@ -192,6 +192,18 @@ export class Generator {
       nodes => t.callExpression(createCallee(), nodes));
   }
 
+  emitVoidExpression(types: TypesDomain, values: ValuesDomain, args: Array<Value>, buildNode_: AbstractValueBuildNodeFunction | BabelNodeExpression): UndefinedValue {
+    this.body.push({
+      args,
+      buildNode: (nodes: Array<BabelNodeExpression>) => t.expressionStatement(
+        (buildNode_: any) instanceof Function ?
```

Comment:
Hmm this particular ternary expression is only used in derive and emitVoidExpression in this file...do you still think its work making a helper function for the two instances or am I overlooking some other use cases?